xbmc/pvr/PVRSettings.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ *      Copyright (C) 2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "PVRSettings.h"
+
+#include "guilib/LocalizeStrings.h"
+#include "utils/StringUtils.h"
+
+using namespace PVR;
+
+void CPVRSettings::MarginTimeFiller(
+  const CSetting * /*setting*/, std::vector< std::pair<std::string, int> > &list, int &current, void * /*data*/)
```

Comment:
IIRC there's some platform that doesn't support unnamed parameters, I guess Jenkins will find out.